bootcamp/models.py

```diff
@@ -0,0 +1,62 @@
+from django.db import models
+from django.utils.timezone import datetime
+
+
+# Create your models here.
+class SingletonModel(models.Model):
+    class Meta:
+        abstract = True
+
+    def save(self, *args, **kwargs):
+        self.pk = 1
+        super(SingletonModel, self).save(*args, **kwargs)
+
+    def delete(self, *args, **kwargs):
+        pass
+
+    @classmethod
+    def load(cls):
+        obj, created = cls.objects.get_or_create(pk=1)
+        return obj
+
+
+class Company(SingletonModel):
+    name = models.CharField(max_length=99, default="Incwell Technology")
+    phone = models.IntegerField(default="5200000,5200001")
+    location = models.CharField(max_length=300, null=False, blank=False, default="Arun thapa Chowk")
+
+    def __str__(self):
+        return f'{self.name}'
+
+
+class Skill(models.Model):
+    skill = models.CharField(max_length = 300, null=False, blank=False, default="DevOps")
+
+    def __str__(self):
+        return f'{self.skill}'
+
+
+class Mentor(models.Model):
+    first_name = models.CharField(max_length = 300, null=False, blank=False, default="John")
+    last_name = models.CharField(max_length = 300, null=False, blank=False, default="Doe")
+    photo = models.FileField(upload_to='bootcamp/static/bootcamp/site-data/profile-pictures', blank=True)
+    designation = models.CharField(max_length=300, null=False, blank=False, default="Software Engineer")
+    summary = models.TextField(null=False, blank=False, default="Lorem ipsum")
+    fb_link = models.CharField(max_length=800, null=False, blank=False, default="https://fb.com")
```

Comment:
@shakyasaijal better to use facebook_username rather than using a short name like fb. Don't confuse by using the abbreviation  like addr_name rather use addressName which is more readable. Make consistent same things in overall project.